src/misc.jl

```diff
@@ -14,6 +14,13 @@ function logmvgamma(p::Int, a::Real)
     return res
 end
 
+"""
+    logmvbeta(p::Int, a::Real, b::Real)
+
+Return the logarithm of the multivariate beta function ([DLMF 35.3.7](https://dlmf.nist.gov/35.3#E7)).
```

Comment:
Put this under the references section? See other docstrings for examples.

Reply:
Not quite sure what you mean. I'm just copying the docstring for `logmvgamma` here.

Reply:
I meant the DLMF reference. But that's already how it looks like for `logmvgamma`.